.env.test

```diff
@@ -7,3 +7,4 @@ POSTGRES_PORT=54320
 POSTGRES_USER=root
 SLACK_BOT_USER_O_AUTH_ACCESS_TOKEN=SLACK_BOT_USER_O_AUTH_ACCESS_TOKEN
 SLACK_SIGNING_SECRET=SLACK_SIGNING_SECRET
+GITLAB_URL=https://git.manomano.tech
```

Comment:
To avoid breaking changes (as mentionned by @greg0ire) this MR will reuse the same idea implemented in the other [MR](https://github.com/ManoManoTech/homer/pull/12/commits/002ad19962bcb4ab1b6046b673094726b2e96890).  
I'm waiting for feedbacks on the implementation before backporting it here.  
I will put this MR in draft mode for now